.github/workflows/pr-ci-pipeline.yml

```
@@ -9,8 +9,43 @@ on:
+        env:
+          MINIO_ROOT_USER: minioadmin
+          MINIO_ROOT_PASSWORD: minioadmin
+        ports:
+          - 9000:9000
+          - 9001:9001
+        options: >-
+          --health-cmd="curl -f http://localhost:9000/minio/health/live || exit 1"
+          --health-interval=10s
+          --health-timeout=5s
+          --health-retries=5
+        command: server /data --console-address ":9001"
```

actionlint (1.7.4)

unexpected key "command" for "services" section. expected one of "credentials", "env", "image", "options", "ports", "volumes"

(syntax-check)